ci/deploy_app.sh

```diff
@@ -0,0 +1,101 @@
+HTTP_KEEP_ALIVE="${HTTP_KEEP_ALIVE:=650}"
+
+EQ_ENABLE_SECURE_SESSION_COOKIE="${EQ_ENABLE_SECURE_SESSION_COOKIE:=True}"
+EQ_RABBITMQ_ENABLED="${EQ_RABBITMQ_ENABLED:=False}"
+EQ_ENABLE_HTML_MINIFY="${EQ_ENABLE_HTML_MINIFY:=False}"
+EQ_RABBITMQ_HOST="${EQ_RABBITMQ_HOST:=rabbit}"
+EQ_RABBITMQ_HOST_SECONDARY="${EQ_RABBITMQ_HOST_SECONDARY:=rabbit}"
+EQ_QUESTIONNAIRE_STATE_TABLE_NAME="${EQ_QUESTIONNAIRE_STATE_TABLE_NAME:=questionnaire-state}"
+EQ_SESSION_TABLE_NAME="${EQ_SESSION_TABLE_NAME:=eq-session}"
+EQ_USED_JTI_CLAIM_TABLE_NAME="${EQ_USED_JTI_CLAIM_TABLE_NAME:=used-jti-claim}"
+EQ_STORAGE_BACKEND="${EQ_STORAGE_BACKEND:=datastore}"
+EQ_SUBMISSION_BACKEND="${EQ_SUBMISSION_BACKEND:=gcs}"
+EQ_FEEDBACK_BACKEND="${EQ_FEEDBACK_BACKEND:=gcs}"
+EQ_PUBLISHER_BACKEND="${EQ_PUBLISHER_BACKEND:=pubsub}"
+EQ_SUBMISSION_CONFIRMATION_BACKEND="${EQ_SUBMISSION_CONFIRMATION_BACKEND:=cloud-tasks}"
+EQ_FULFILMENT_TOPIC_ID="${EQ_FULFILMENT_TOPIC_ID:=eq-fulfilment-topic}"
+EQ_INDIVIDUAL_RESPONSE_LIMIT="${EQ_INDIVIDUAL_RESPONSE_LIMIT:=75}"
+EQ_INDIVIDUAL_RESPONSE_POSTAL_DEADLINE="${EQ_INDIVIDUAL_RESPONSE_POSTAL_DEADLINE:=2021-04-28T02:00:00+00:00}"
+EQ_FEEDBACK_LIMIT="${EQ_FEEDBACK_LIMIT:=10}"
+EQ_GCS_SUBMISSION_BUCKET_ID="${PROJECT_ID}-submission"
+EQ_GCS_FEEDBACK_BUCKET_ID="${PROJECT_ID}-feedback"
+COOKIE_SETTINGS_URL="${COOKIE_SETTINGS_URL:=}"
+CDN_URL="${CDN_URL:=https://cdn.census.gov.uk}"
+CDN_ASSETS_PATH="${CDN_ASSETS_PATH:=/design-system}"
+ADDRESS_LOOKUP_API_URL="${ADDRESS_LOOKUP_API_URL:=}"
+ADDRESS_LOOKUP_API_AUTH_ENABLED="${ADDRESS_LOOKUP_API_AUTH_ENABLED:=False}"
+ADDRESS_LOOKUP_API_AUTH_TOKEN_LEEWAY_IN_SECONDS="${ADDRESS_LOOKUP_API_AUTH_TOKEN_LEEWAY_IN_SECONDS:=10}"
+EQ_NEW_RELIC_ENABLED="${EQ_NEW_RELIC_ENABLED:=False}"
+CONFIRMATION_EMAIL_LIMIT="${CONFIRMATION_EMAIL_LIMIT:=10}"
+
+EQ_GOOGLE_TAG_MANAGER_ID="${EQ_GOOGLE_TAG_MANAGER_ID:=}"
+EQ_GOOGLE_TAG_MANAGER_AUTH="${EQ_GOOGLE_TAG_MANAGER_AUTH:=}"
+NEW_RELIC_APP_NAME="${NEW_RELIC_APP_NAME:=}"
+NEW_RELIC_LICENSE_KEY="${NEW_RELIC_LICENSE_KEY:=}"
+
+
+gcloud beta run deploy eq-questionnaire-runner \
+    --project="${PROJECT_ID}" --region="${REGION}" --concurrency="${CONCURRENCY}" --min-instances="${MIN_INSTANCES}" --max-instances="${MAX_INSTANCES}" \
+    --port=5000 --cpu="${CPU}" --memory="${MEMORY}" \
+    --image="${DOCKER_REGISTRY}/eq-questionnaire-runner:${IMAGE_TAG}" --platform=managed --allow-unauthenticated \
+    --vpc-connector="redis-vpc" \
+    --service-account="cloud-run@${PROJECT_ID}.iam.gserviceaccount.com" \
+    --set-secrets EQ_REDIS_HOST="redis-host:latest" \
+    --set-secrets EQ_REDIS_PORT="redis-port:latest" \
+    --set-secrets "/keys/keys.yml"="keys:latest" \
+    --set-secrets "/secrets/secrets.yml"="secrets:latest" \
+    --set-env-vars EQ_STORAGE_BACKEND="${EQ_STORAGE_BACKEND}" \
+    --set-env-vars EQ_ENABLE_SECURE_SESSION_COOKIE="${EQ_ENABLE_SECURE_SESSION_COOKIE}" \
+    --set-env-vars EQ_SECRETS_FILE="/secrets/secrets.yml" \
+    --set-env-vars EQ_KEYS_FILE="/keys/keys.yml" \
```

Comment:
The keys and secrets file paths should be defined as variables above, something like:
```
EQ_KEYS_FILE="/keys/keys.yml"
EQ_SECRETS_FILE="/secrets/secrets.yml"
```

and referenced here:
```
--set-secrets "${EQ_KEYS_FILE}"="keys:latest" \
--set-secrets "${EQ_SECRETS_FILE}"="secrets:latest" \
--set-env-vars EQ_KEYS_FILE="${EQ_KEYS_FILE}" \
--set-env-vars EQ_SECRETS_FILE="${EQ_SECRETS_FILE}" \
```

The env vars should also be straight after the `set-secrets`.

Reply:
Added now, thanks!